Riot/Modules/SideMenu/SideMenuCoordinator.swift

```diff
@@ -215,6 +219,18 @@ final class SideMenuCoordinator: NSObject, SideMenuCoordinatorType {
         
         self.exploreRoomCoordinator = exploreRoomCoordinator
     }
+    
+    private func showMembers(spaceId: String, session: MXSession) {
+        let parameters = SpaceMembersCoordinatorParameters(userSessionsService: parameters.appCoordinator.userSessionsService, session: session, spaceId: spaceId)
+        let spaceMembersCoordinator = SpaceMembersCoordinator(parameters: parameters)
+        spaceMembersCoordinator.delegate = self
+        let presentable = spaceMembersCoordinator.toPresentable()
+        presentable.presentationController?.delegate = self
```

Comment:
Even if it's fine to let it here I think it can be interesting to put `presentable.presentationController?.delegate = self` inside `SpaceMembersCoordinator`. We will add this to the file template. And add a delegate method for this case. It will centralise this edge case and avoid to forget it for other screens.